README.md

```diff
@@ -12,6 +15,7 @@ A library for avoiding TransactionTooLargeException during state saving and rest
 * [Limitations](#limitations)
 * [Testing](#testing)
 * [License](#license)
+* [Javadoc](https://jitpack.io/com/github/livefront/bridge/feature~hosted-javadoc-v1.1.0-g45d8d59-1/javadoc/index.html)
```

Comment:
Hmm interesting....is there anyway to _not_ have to do this manually. This seems error prone. At least the fact that we have to specify `feature~hosted-javadoc-v1.1.0-g45d8d59-1` as opposed to `feature~hosted-javadoc-v1.1.0` for example, because that means we won't know the value until _after_ we release right?